stacker/src/arena_hashmap.rs

```diff
@@ -235,6 +235,29 @@ impl ArenaHashMap {
         }
     }
 
+    #[inline]
+    /// This will invalidate the hashmaps get and insert methods.
+    /// Only iter() is still valid afterwards
+    ///
+    /// # Safety
+    /// Any call to get or mutate_or_create after this call is undefined behavior.
+    pub unsafe fn iter_mut_keys<F: Fn(&mut [u8])>(&mut self, cb: F) {
```

Comment:
Could this also be iterator-based, i.e. return `impl Iterator<Item=&mut [u8]>` or do the lifetimes not work out?

Reply:
Yes, lifetimes on mutable iterators are really messy and not worth here